### PR TITLE
Create Redux files

### PR DESCRIPTION
**Pull request by fibini**
Create Redux files using es6 syntax and ducks pattern.

### CODE REVIEW

File: src/redux/books/books.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+const ADD_BOOK = 'ADD_BOOK';
+const REMOVE_BOOK = 'REMOVE_BOOK';
```
Comment (enkog) [inline src/redux/books/books.js]: - [ ] Please modify the ADD_BOOK and REMOVE_BOOK action types to match the ducks pattern in order to complete this project's requirement. 
- The new versions should be something like this:

  `const ADD_BOOK = bookstore/books/ADD_BOOK`
  `const REMOVE_BOOK= bookstore/books/REMOVE_BOOK`

File: src/redux/categories/categories.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+const CHECKED_STATUS = 'CHECKED_STATUS';
```
Comment (enkog) [inline src/redux/categories/categories.js]: - [ ] Also modify the CHECKED_STATUS action type to match the ducks pattern in order to complete this project's requirement.
 - **Hint**: `const CHECK_STATUS = bookstore/categories/CHECK_STATUS`

**Review by enkog — CHANGES_REQUESTED** [review]
Hi @fibini 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Required Changes ♻️

- [ ] On the **Getting Started** section of your `README.md`, consider including detailed and formatted instructions to improve professionalism. It would be nice to have each instruction on a single line.

_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Davidosky007 — APPROVED** [review]
Hi @fibini 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
* No linter errors 👍 
* Well written readme file 👍 
* Proper use of Gitflow 👍 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**